### PR TITLE
feat: [OENG-12871]: Create dymamic links for Error Tracking

### PR DESCRIPTION
**Pull request by DFieldFL**
##### Summary:

Error Tracking dynamic links

##### Jira Links:

https://harness.atlassian.net/browse/OENG-12871

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Review by ccaspanello — DISMISSED** [review]

File: src/modules/85-cv/components/LogsAnalysis/components/LogAnalysisRow/LogAnalysisRow.tsx [hunk 1]
```diff
@@ -58,7 +58,36 @@ function DataRow(props: LogAnalysisDataRowProps): JSX.Element {
   const [displayRiskEditModal, setDisplayRiskEditModal] = useState(false)
   const [feedbackGiven, setFeedbackGiven] = useState<{ risk: string; message: string } | undefined>(undefined)
   const logTextRef = useRef<HTMLParagraphElement>(null)
-  const onShowRiskEditModalCallback = useCallback(() => setDisplayRiskEditModal(true), [])
+  const onShowRiskEditModalCallback = useCallback(() => {
```
Comment (anjanBalgovind) [inline src/modules/85-cv/components/LogsAnalysis/components/LogAnalysisRow/LogAnalysisRow.tsx:61]: can this moved out to a utils file

File: src/modules/85-cv/components/ExecutionVerification/components/ExecutionVerificationSummary/components/SummaryOfDeployedNodes/SummaryOfDeployedNodes.tsx [hunk 2]
```diff
@@ -77,6 +81,14 @@ export function SummaryOfDeployedNodes(props: SummaryOfDeployedNodesProps): JSX.
           titleText={getString('pipeline.verification.logClustersInViolation')}
           tabId={getString('pipeline.verification.analysisTab.logs')}
         />
+        {isErrorTrackingEnabled && (
```
Comment (anjanBalgovind) [inline src/modules/85-cv/components/ExecutionVerification/components/ExecutionVerificationSummary/components/SummaryOfDeployedNodes/SummaryOfDeployedNodes.tsx:84]: center this content on the screen with the other summary texts

File: src/modules/85-cv/components/LogsAnalysis/components/LogAnalysisRow/LogAnalysisRow.tsx (hunk 1 above)
Comment (DFieldFL) [reply src/modules/85-cv/components/LogsAnalysis/components/LogAnalysisRow/LogAnalysisRow.tsx:61]: 👍

File: src/modules/85-cv/components/ExecutionVerification/components/ExecutionVerificationSummary/components/SummaryOfDeployedNodes/SummaryOfDeployedNodes.tsx (hunk 2 above)
Comment (DFieldFL) [reply src/modules/85-cv/components/ExecutionVerification/components/ExecutionVerificationSummary/components/SummaryOfDeployedNodes/SummaryOfDeployedNodes.tsx:84]: Looks like they all are left aligned 
image

**Review by ccaspanello — APPROVED** [review]

**Comment by DFieldFL** [comment]
retrigger jest

**Comment by DFieldFL** [comment]
retrigger jest

**Review by anjanBalgovind — APPROVED** [review]

**Pull request merged**